app/controllers/avo/search_controller.rb

```diff
@@ -61,13 +63,21 @@ def search_resource(resource)
       result_object = {
         header: header,
         help: resource.fetch_search(:help) || "",
-        results: results,
+        results: highlight_search_results(results, params[:q]),
```

Comment:
Instead of applying the highlight here and doing another loop, can't we do it straight where we write the initial object?

https://github.com/avo-hq/avo/pull/2370/files#diff-1a43067095b17211ecd0b229a1ad7bc86cc97b3d16a7044d78a2a0455463f2c6R139

Reply:
Yes, we can do that. I will update it. Thanks